docs/app/Examples/elements/List/Variations/ListAnimatedExample.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+import {List, ListItem, Message} from 'stardust';
+
+export default class ListAnimatedExample extends Component {
+  render() {
+    const avatar1 = <img className='ui avatar image' src='http://lorempixel.com/100/100/people/1' />;
+    const avatar2 = <img className='ui avatar image' src='http://lorempixel.com/100/100/people/2' />;
+    const avatar3 = <img className='ui avatar image' src='http://lorempixel.com/100/100/people/3' />;
+    return (
+      <div>
+        <Message className='info'>
+        Be sure content can fit on one line, otherwise text content will reflow when hovered.
+        </Message>
```

Comment:
This file is rendered on the doc site under the example.  To keep it clean, put the Message as a child of the `< ComponentExample />` in `ListVariationsExamples.js`.

The `<ComponentExample />` was extended to render children for this purpose.